apps/backend/src/db/migrations/20250816130844_alter_files_table.ts

```diff
@@ -0,0 +1,28 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = {
+	FILES: "files",
+};
+
+const ColumnName = {
+	ENTITY_ID: "entity_id",
+	FOLDER: "folder",
+} as const;
+
+const FOLDER_VALUES = ["avatars", "pois", "reviews", "routes"];
+
+async function down(knex: Knex): Promise<void> {
+	await knex.schema.alterTable(TABLE_NAME.FILES, (table) => {
+		table.dropColumn(ColumnName.FOLDER);
+		table.dropColumn(ColumnName.ENTITY_ID);
+	});
+}
+
+async function up(knex: Knex): Promise<void> {
+	await knex.schema.alterTable(TABLE_NAME.FILES, (table) => {
+		table.integer(ColumnName.ENTITY_ID).notNullable();
+		table.enu(ColumnName.FOLDER, FOLDER_VALUES).notNullable();
+	});
```

Comment:
We have already files inserted in the files table on environment, we can't just set new columns to nonNullable easily

FileFolderName in the example should be created in the migration, not imported from the shared

```suggestion
	await knex.schema.alterTable(TABLE_NAME.FILES, (table) => {
		table.integer(ColumnName.ENTITY_ID).defaultTo(0).notNullable();
		table.enum(ColumnName.FOLDER, FOLDER_VALUES).defaultTo(FileFolderName.AVATARS).notNullable();
	});
```